### PR TITLE
making a Portable FM Radio Station

### PR DESCRIPTION
**Pull request by AashutoshR2062**
## Describe your project in 1-3 sentences

I am building a portable FM station which is a small, portable radio station that is powered by a Raspberry Pi. It can be used to transmit an FM radio signal, allowing users to broadcast their own radio show or play music over a small area.

## Hack Club Slack handle

Aashutosh Ravi

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by Arpan-206** [comment]
This needs a lot of regulatory approvals if you are in India, which I'm assuming from the name. So I'd recommend checking if you can transmit.

**Comment by AashutoshR2062** [comment]
> This needs a lot of regulatory approvals if you are in India, which I'm assuming from the name. So I'd recommend checking if you can transmit.


Thanks, @Arpan-206   for your concern about the regulatory approvals.
But i am from Nepal and a mini FM station project is used for
an educational demonstration that doesn't cause interference with others 
the radio station will not require regulatory approvals. So one can 
transmit the radio signal within a short range(70m).

**Comment by Arpan-206** [comment]
> > This needs a lot of regulatory approvals if you are in India, which I'm assuming from the name. So I'd recommend checking if you can transmit.
> 
> 
> Thanks, @Arpan-206   for your concern about the regulatory approvals.
> But i am from Nepal and a mini FM station project is used for
> an educational demonstration that doesn't cause interference with others 
> the radio station will not require regulatory approvals. So one can 
> transmit the radio signal within a short range(70m).
> 


Great then, wonderful project.

File: AashutoshR2062.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+name: "@AashutoshR2062"
+project: "Portable FM Radio Station"
+---
+
+# Project Name
```
Comment (bellesea) [inline AashutoshR2062.md:6]:
```suggestion
# Portable FM radio Staion
```

File: AashutoshR2062.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Portable FM radio Staion
```
Comment (bellesea) [inline AashutoshR2062.md:7]:
```suggestion
```

**Review by bellesea — CHANGES_REQUESTED** [review]
@AashutoshR2062 this is super cool! do you have a go to radio station?

a few questions:
- what do you mean by "will be able to play custom sound"?
- are you following any sort of guide or referencing an existing project for this?
- which FM station software will you be downloading?
- if you're connecting your pi to a monitor do you also have a keyboard and mouse? or were you thinking of a headless pi set up?
- how would you connect your antenna to your pi?

thank you sm!

**Comment by AashutoshR2062** [comment]
No, I don't have any go to the radio station.

I mean by "will be able to play custom sound" that we can add our custom music to play or live sound via mic. Yes, I am being guided by a teacher for this project. 

I will be downloading the FM transmitter software from the GitHub page https://github.com/markondej/fm_transmitter

Yes, I do have a keyboard and mouse. I am not thinking about the headless pi setup. First we will require the keyboard, mouse, and monitor to set up the PI, after that we will be able to run the station. 

I will connect my antenna via jumper wire in GPIO pin 4 of Raspberry Pi

**Comment by AashutoshR2062** [comment]
> @AashutoshR2062 this is super cool, i'm wondering... how do you know that the FM antenna would work with and connect to the jumper wire? also, do you have the jumper wire already? and, how do you plan on programming your pi? do you have a monitor/mouse/etc. or will you be doing a headless set up and have an SD card reader...

GPIO pin 4 of Raspberry Pi is used by the software to emit the signals. Yeah, I have a jumper wire for the antenna whose one end will be connected to the pin and the other end wired up in the Antenna's end.

So fisrt I will set up Raspberry Pi OS and then clone a script for broadcasting the audio from the microphone.

Script link: https://github.com/markondej/fm_transmitter

I already have all the materials required to set up pi for the project like a mouse, keyboard, monitor, and sd card reader.

**Review by bellesea — APPROVED** [review]
@AashutoshR2062 this is awesome and can't wait to see you work on this :))

a fabulous member of @hackclub/bank-ops will be reaching out with next steps!

btw, approving to move this forward but could you please commit the github repo you linked within your file? thank you :)

**Review by hackliv — APPROVED** [review]
Hey @AashutoshR2062 this looks so cool!! I'm dming you now on Slack to chat about next steps for the grant. :)

**Pull request merged**